frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts

```diff
@@ -0,0 +1,106 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { untilNoLoadersPresent } from '@console/internal-integration-tests/views/crud.view';
+import { waitFor } from '@console/ceph-storage-plugin/integration-tests/utils/helpers';
+import {
+  efficiencyValue,
+  healthOfMCG,
+  noobaaCount,
+  obcCount,
+  resourceProviders,
+  savingsValue,
+  resiliencyOfMCG,
+} from '../views/noobaaDashboardPage.view';
+import { testBucket } from '../mocks/obcData';
+
+const loadAndWait = async () => {
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+describe('Tests Buckets Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Test at least one noobaa bucket is present', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
```

Comment:
Since this is already checked in beforeAll, why do you need to check it again here?

Reply:
Its present at other places too

Reply:
yes. For some reason, it fails without this.